modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -8,7 +8,7 @@
 
 An {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortname} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.
 
-`ClusterTask` functionality is deprecated since {pipelines-shortname} 1.10 and is planned for removal in a future release. If your pipelines use `ClusterTasks`, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the existing `ClusterTasks`.
+Until version 1.16, {pipelines-shortname} included `ClusterTask` functionality. Versions 1.17 and later no longer include this functionality. If your pipelines use `ClusterTask` references, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the previously existing `ClusterTask` definitions.
```

Comment:
Is it `you can re-create them with the tasks` or should we say `you can update them with the tasks`

Reply:
About "update them" - I am really not sure how you can update a pipeline in place? If you *can* do that, let's make it "update or re-create them"

About "renaming the file" - I don't think so. it's actually "op-resolver-cluster", the name of the cluster resolver file, plus "-tasks-ref".

Reply:
> About "renaming the file" - I don't think so. it's actually "op-resolver-cluster", the name of the cluster resolver file, plus "-tasks-ref".